src/main/java/uk/gov/hmcts/reform/prl/services/CaseWorkerEmailService.java

```diff
@@ -0,0 +1,129 @@
+
+    @Autowired
+    private EmailService emailService;
+
+    @Value("${uk.gov.notify.email.application.email-id}")
+    private  String courtEmail;
+
+    @Value("${uk.gov.notify.email.application.court-name}")
+    private  String courtName;
+
+    @Value("${xui.url}")
+    private String manageCaseUrl;
+
+    public EmailTemplateVars buildEmail(CaseDetails caseDetails, UserDetails userDetails) {
+
+        List<PartyDetails> applicants = caseDetails.getCaseData()
+            .getApplicants()
+            .stream()
+            .map(Element::getValue)
+            .collect(Collectors.toList());
+
+        List<String> applicantNamesList = applicants.stream()
+            .map(element -> element.getFirstName() + " " + element.getLastName())
+            .collect(Collectors.toList());
+
+        List<PartyDetails> respondents = caseDetails.getCaseData()
+            .getRespondents()
+            .stream()
+            .map(Element::getValue)
+            .collect(Collectors.toList());
+
+        List<String> respondentsList = respondents.stream()
+            .map(PartyDetails::getLastName)
+            .collect(Collectors.toList());
+
+        String caseUrgency = null;
+
+        if (caseDetails.getCaseData().getIsCaseUrgent().equals(YesOrNo.YES)) {
+            caseUrgency = URGENT_CASE;
+        } else if (caseDetails.getCaseData().getDoYouNeedAWithoutNoticeHearing().equals(YesOrNo.YES)) {
+            caseUrgency = WITHOUT_NOTICE;
+        } else if ((caseDetails.getCaseData().getIsCaseUrgent().equals(YesOrNo.NO))
+                && (caseDetails.getCaseData().getDoYouNeedAWithoutNoticeHearing().equals(YesOrNo.YES))
+                && (caseDetails.getCaseData().getDoYouRequireAHearingWithReducedNotice().equals(YesOrNo.NO))) {
+            caseUrgency = STANDARAD_HEARING;
+        }
+
+        final String[] typeOfOrder = new String[1];
+
+        caseDetails.getCaseData().getOrdersApplyingFor().forEach(orderType -> {
+            if (orderType.equals(OrderTypeEnum.childArrangementsOrder)) {
+                typeOfOrder[0] = OrderTypeEnum.childArrangementsOrder.getDisplayedValue();
+            } else if (orderType.equals(OrderTypeEnum.prohibitedStepsOrder)) {
+                typeOfOrder[0] = OrderTypeEnum.prohibitedStepsOrder.getDisplayedValue();
+            } else {
+                typeOfOrder[0] = OrderTypeEnum.specificIssueOrder.getDisplayedValue();
+            }
+        });
+
+        String applicantNames = String.join(", ", applicantNamesList);
+        String respondentNames = String.join(", ", respondentsList);
+
+        EmailTemplateVars emailTemplateVars = CaseWorkerEmail.builder()
+            .caseReference(caseDetails.getCaseId())
+            .caseName(caseDetails.getCaseData().getApplicantCaseName())
+            .applicantName(applicantNames)
+            .respondentLastName(respondentNames)
+            .hearingDateRequested("  ")
+            .ordersApplyingFor(typeOfOrder[0])
```

Comment:
Will this work if there are multiple orders are selected? ignore if OrdersApplyingFor is a radio button

Reply:
Yes. This will work because, in the above we are looping through the list of OrdersApplyingFor.